test/e2e/functional/tests/e2e.go

```diff
@@ -183,12 +184,12 @@ var _ = Describe("metallb", func() {
 
 			assertConfigMapMatchYAML("{}")
 		},
-			table.Entry("Test AddressPool object with default auto assign", "addresspool1", &metallbv1beta1.AddressPool{
+			table.Entry("Test AddressPool object with default auto assign", "addresspool1", &v1beta1.AddressPool{
```

Comment:
These tests will need to be removed, as there's no configmap anymore.
We may want to replace them by extending (at least) the unit tests in metallb to ensure we are doing the cr handling correctly.

Reply:
yes @fedepaol , removed all of the references relevant to address pool, bgp peer and bfd profile crd's.